SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -83,3 +113,128 @@ FReply FSpatialGDKEditorLayoutDetails::ClickedOnButton()
 
 	return FReply::Handled();
 }
+
+bool FSpatialGDKEditorLayoutDetails::TryConstructMobileCommandLineArgumentsFile(FString& CommandLineArgsFile)
```

Comment:
It looks like this is always constructing the command line args as new, not adding to something already there. It might be cleaner (and possibly very marginally faster) to return a string and check for it being empty.

Reply:
this is because we are always deleting the file after pushing the command line arguments. Would you prefer for the file to stay around?